force-app/main/default/objects/Journey_Task__c/validationRules/Relative_days_to_complete_restriction.validationRule-meta.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Relative_days_to_complete_restriction</fullName>
+    <active>true</active>
+    <description>This ensures that the values for the days are not negative numbers.</description>
+    <errorConditionFormula>Relative_Days_to_Complete__c &lt; 1</errorConditionFormula>
+    <errorMessage>INVALID VALUE</errorMessage>
```

Comment:
Validation rules should provide a concise and useful error message to help the end users understand why the record isn't being saved. `INVALID VALUE` doesn't provide any information.

The error message should be a brief description of the formula used in the validation rule:
> Relative Days to Complete cannot be less than 1 day.